pipeline_steps/common.groovy

```diff
@@ -729,7 +729,10 @@ void use_node(String label=null, body){
                          env.RPC_GATING_BRANCH)
       install_ansible()
       print "${env.NODE_NAME} preparation complete, now ready for use."
-      body()
+      // Ensure log size is not excessive
```

Comment:
Why is this needed here, why does `globalWraps` not suffice?

Reply:
Ooops, that was my first attempt before I added globalwraps. Will remove.

Reply:
Done